inyoka_theme_default/static/style/inyoka/portal.less

```diff
@@ -54,9 +54,159 @@
       }
     }
   }
+
   &-usericon {
     font-size:  20ex;
     text-align: center;
     vertical-align: middle;
   }
+
+  &-privmsg-table {
+    th.actions {
+      text-align: center;
+    }
+    .actions {
+      text-align: right;
+    }
+    th.delete {
+      text-align: center;
+      input {
+        margin-left: 5px;
+      }
+    }
+    .pn_delete {
+      text-align: center;
+    }
+
+    @media (min-width: @screen-md-min) {
+      .actionscol {width:10%;}
+      .deletecol {width:14%;}
+    }
+    @media (min-width: @screen-lg-min) {
+      .actionscol {width:10%;}
+      .deletecol {width:12%;}
+    }
+
+    @media (max-width: @screen-xs-max) {
+      thead {display:none;}
+      tbody {
+        tr {
+          display:block;
+          width:100%;
+          height:auto;
+          border-top:1px solid @gray-lighter;
+          margin:15px 0 25px;
+        }
+        td {
+          display: block;
+          float:left;
+          width:50%;
+          border:0;
+        }
+        td:before {
+          display: inline-block;
+          width: auto;
+          padding-right: 10px;
+          white-space: nowrap;
+          content: attr(data-title)':';
+        }
+        td.action,
+        td.pn_delete {
+          margin-bottom: 25px;
+        }
+      }
+      tfoot > tr {
+        > td {
+          border-top: none;
+        }
+      }
+    }
+  }
+  .message {
+    margin-bottom: 25px;
+    .head {
+      background: @gray-lighter;
+      padding:10px 8px;
+      border: 1px solid #ddd;
+      @media (max-width: @screen-xs-max) {
+        position: relative;
+      }
+      {
```

Comment:
less does not want to compile because of this block. Is it even needed as there are no `<dd>`/`<dt>` anymore?